src/errors.rs

```diff
@@ -104,11 +104,10 @@ fn test_source() {
 #[test]
 #[cfg(feature = "safe_api")]
 fn test_unknown_crypto_from_decode_error() {
-    use base64::DecodeError;
+    use ct_codecs::Error;
 
-    let err_one = DecodeError::InvalidByte(0, 0);
-    let err_two = DecodeError::InvalidLastSymbol(0, 0);
-    let err_three = DecodeError::InvalidLength;
+    let err_one = Error::InvalidInput;
+    let err_two = Error::Overflow;
 
     // Tests Debug impl through "{:?}" and Display impl though "{}"
```

Comment:
This comment confused me. Are we still testing both `Debug` and `Display`, or is it just `Display` now?

Reply:
We're still testing both. It's just that in the later `format!` call, the note explains which formatter is used to test which impl, if that's makes sense.

Reply:
Oops! For some reason, I thought that the format string had changed. I see it now.